### ISSUE TITLE
ip_to_geojson does not return JSON

### ISSUE THREAD

**Issue by blackfist** [yolothreat/utilitybelt#56]
WTH?

**Comment by krmaxwell** [comment]
I welcome your PR.

**Comment by jeffbryner** [comment]
Apparently it once did: https://github.com/yolothreat/utilitybelt/commit/5b749c2600a0777055718374e8f3314500620f96

**Comment by blackfist** [comment]
So we need to ask @sroberts why that was changed.

**Comment by sroberts** [comment]
So this is something I've been back and forth on. The idea of Utilitybelt is a Python library used within a other python apps (primarily). So if it's being called as a library doesn't it make more sense to respond with a Python native data structure rather than an intermediate format? I found myself writing a tool in Python, that called UB, got back JSON, and parsed the JSON back into a dict. It seemed unnecessary.

**Comment by jeffbryner** [comment]
Agreed, but if it says 'Generate GeoJSON for given IP address' it should return json. 

Maybe the 'right' way is to have two functions called by a wrapper..one returns a dict, one json

**Comment by krmaxwell** [comment]
I like @jeffbryner 's suggestion, so that we're unambiguous about the returned data.

**Comment by sroberts** [comment]
I couldn't agree more. There are certainly use cases for both, they just need to be explicit. I dig it. :+1:

**Comment by krmaxwell** [comment]
that sounds suspiciously like "we welcome your PR" :D :D :D
